### ISSUE TITLE
Split https://lima-vm.io/docs/config/network/ to multiple pages

### ISSUE THREAD

**Issue by AkihiroSuda** [lima-vm/lima#1815]
https://lima-vm.io/docs/config/network/ is too long and hard to follow.

It is also unclear which network is the best.
(vzNAT for host-to-guest, user-v2 for guest-to-guest, bridged for other-host-to-guest)

**Comment by ayushgml** [comment]
Hey! Can i work on this issue?

**Comment by jandubois** [comment]
> Hey! Can i work on this issue?

Absolutely! Thank you for offering!

**Comment by ayushgml** [comment]
Great! then i will get started.

**Comment by ayushgml** [comment]
Hey @jandubois ! I have been trying to find the document to edit but it shows 404 status to me when i go and click on 'Edit this page':

<img width="621" alt="Screenshot 2023-10-13 at 12 27 57 PM" src="https://github.com/lima-vm/lima/assets/72748253/7e330a56-71fc-460f-88be-32bdadf8152a">

<img width="1334" alt="Screenshot 2023-10-13 at 12 19 06 PM" src="https://github.com/lima-vm/lima/assets/72748253/67c7b854-16ff-47d7-a5b8-29f0b0ffd82a">

Can you help me out on this?

**Comment by AkihiroSuda** [comment]
The file is in https://github.com/lima-vm/lima/tree/master/website/content/en/docs/Config/Network

**Comment by AkihiroSuda** [comment]
@ayushgml Do you still plan to work on this?

**Comment by kairveeehh** [comment]
> The file is in https://github.com/lima-vm/lima/tree/master/website/content/en/docs/Config/Network

hii , can you let me know how the content needs to divided like just under headings or smt specific , also is this still up for work?

**Comment by AkihiroSuda** [comment]
<https://lima-vm.io/docs/config/network/> should be just split to the following pages, without changing the contents:
- `docs/config/network/user`: Default user-mode network (192.168.5.0/24)
- `docs/config/network/user-v2`: Lima user-v2 network 
- `docs/config/network/vmnet`: VMNet networks


The flowchart can remain in <https://lima-vm.io/docs/config/network/>

**Comment by kairveeehh** [comment]
> https://lima-vm.io/docs/config/network/ should be just split to the following pages, without changing the contents:
> 
> * `docs/config/network/user`: Default user-mode network (192.168.5.0/24)
> * `docs/config/network/user-v2`: Lima user-v2 network
> * `docs/config/network/vmnet`: VMNet networks
> 
> The flowchart can remain in https://lima-vm.io/docs/config/network/

I worked on this it is done but for some reason , the git changes are being huge , even though there are no differences in the working directory if I check it shows 46 unchanged files as changed while even on discarding they re-appear , is it normal?
